README.md

```diff
@@ -40,32 +40,21 @@ The [Authentication template](https://www.npmjs.com/package/@brightlayer-ui/cra-
 
 ## Usage
 
-You can use these templates with the Brightlayer UI CLI (recommended):
+You can use these templates with the Brightlayer-UI NPM starter package:
 
 ```sh
-npx -p @brightlayer-ui/cli blui new react --template=blank-typescript
-npx -p @brightlayer-ui/cli blui new react --template=routing-typescript
-npx -p @brightlayer-ui/cli blui new react --template=authentication-typescript
-```
-
-or with the Create React App CLI:
-
-```sh
-npx create-react-app app-name --template @brightlayer-ui/blank-typescript
-npx create-react-app app-name --template @brightlayer-ui/routing-typescript
-npx create-react-app app-name --template @brightlayer-ui/authentication-typescript
+npx -p @brightlayer-ui/create-blui-react-app
```

Comment:
Don't you need to specify the command here after the package? Or should we remove the -p flag since the command is the same as the package (not sure if that works for a scoped package).